### PR TITLE
Add VC-A11 ETH hodler

### PR DESCRIPTION
**Pull request by zTgx**
as title.

### CODE REVIEW

File: tee-worker/litentry/core/assertion-build/src/a11.rs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+			credential_unsigned.credential_subject.values.push(flag);
+
+			return Ok(credential_unsigned)
+		},
+		Err(e) => {
+			error!("Generate unsigned credential failed {:?}", e);
+		},
+	}
+
+	Err(Error::Assertion7Failed)
```
Comment (Kailai-Wang) [inline tee-worker/litentry/core/assertion-build/src/a11.rs]: Here should be `Assertion11Failed`

File: tee-worker/litentry/core/assertion-build/src/a11.rs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+use std::{
+	str::from_utf8,
+	string::{String, ToString},
+	vec,
+	vec::Vec,
+};
+
+pub fn build(
+	identities: Vec<Identity>,
+	min_balance: u128,
```
Comment (Kailai-Wang) [inline tee-worker/litentry/core/assertion-build/src/a11.rs]: Shall we use the type `Balance` here?

**Review by Kailai-Wang — COMMENTED** [review]
Thanks, two small things:

File: tee-worker/litentry/core/assertion-build/src/a11.rs (hunk 1 above)
Comment (zTgx) [reply tee-worker/litentry/core/assertion-build/src/a11.rs]: Thanks, fixed.

File: tee-worker/litentry/core/assertion-build/src/a11.rs (hunk 2 above)
Comment (zTgx) [reply tee-worker/litentry/core/assertion-build/src/a11.rs]: Yes, Balance is better.

**Review by jingleizhang — APPROVED** [review]
LGTM

**Pull request merged**